src/frontend/components/node--agency--card.tsx

```diff
@@ -1,8 +1,16 @@
 import Image from "next/image";
-import { DrupalNode } from "next-drupal";
 
 interface NodeAgencyCardProps {
-  node: DrupalNode;
```

Comment:
I had to update this because DrupalNode type described `path` as a DrupalPathAlias which wouldn't work since the href needed a string.